clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/input/kafka/VeniceKafkaInputMapper.java

```diff
@@ -24,12 +28,23 @@ protected AbstractVeniceRecordReader<BytesWritable, KafkaInputMapperValue> getRe
     throw new UnsupportedOperationException();
   }
 
+  @Override
+  protected Optional<AbstractVeniceFilter<KafkaInputMapperValue>> getFilter(final VeniceProperties props) {
```

Comment:
IMO `final` in method argument without strong reason does not add much value. Rather they increase the mental burden.

Reply:
I believe there are many debates on this already so not going to expand them here. In this method, as `VeniceProperties` is read-only, declaring it as `final` doesn't add much value indeed. I guess I just have this habit to declare anything I don't want to change as `final`. Do we have code convention on this?

Reply:
Nope, we don't. But except for a few cases, I don't think we mark method args as final. 


> In this method, as VeniceProperties is read-only, declaring it as 
It can still be updated though